be/src/storage/rowset/beta_rowset_writer.cpp

```diff
@@ -278,7 +346,11 @@ Status HorizontalBetaRowsetWriter::_flush_chunk(const vectorized::Chunk& chunk)
         _num_rows_written += static_cast<int64_t>(chunk.num_rows());
         _total_row_size += static_cast<int64_t>(chunk.bytes_usage());
     }
-    return _flush_segment_writer(&segment_writer.value());
+    if (seg_info) {
+        seg_info->set_num_rows(static_cast<int64_t>(chunk.num_rows()));
+        seg_info->set_row_size(static_cast<int64_t>(chunk.bytes_usage()));
```

Comment:
should be update, not set?

Reply:
flush chunk means one chunk as segment, no need update.